### DISCUSSION TITLE
A few ideas for osu!lazer Skin editor

### DISCUSSION THREAD

**Discussion by TempheX** [ppy/osu#30674] [Ideas]
I know that the skin editor is not a priority like Mapping and Gameplay, but I think is something important for people that they want to make their own skin. So i'm just gonna start with some ideas about lazer default Skin elements:

-Be able to change the font of the elements.
-PP counter overlay can start from maximum pp can get to 0. Basically... backwards instead of starting from 0
-Like for mapping now elements can be seen on a folder, this should be the same about skins.

   Speaking of elements, should be great leave a great quantity of many types of circles, hitsounds, etc... in just one folder and select the ones you want for your skin in the Skin Editor... 

   I'm not gonna talk about the interface of the skin editor because I think peppy already mention something he has in mind about change the whole layout to a one where is full of layers(I imagine something like Photoshop). So I don't have opinions about that.
   
   I know this isn't a lot, but this is all I can think right now... Also sorry for my bad english.

**Comment by Joehuu** [comment]
The third point will be addressed in https://github.com/ppy/osu/pull/30226.
